_includes/_components/fixtures-load.html

```diff
@@ -1,50 +1,80 @@
-<!-- TODO -->
-
-      <article class="component blue">
-        <header>
-          <div class="container">
+<article class="component blue">
+    <header>
+        <div class="container">
             <h2>Fixtures Load</h2>
-            <p>This component blablabla</p>
-          </div>
-        </header>
-        <div class="container content">
-          <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-          </p>
-          <div class="row">
+            <p>One command allows quick to load your fake data created with nelmio/alice library.</p>
+        </div>
+    </header>
+
+    <div class="container content">
+        <div class="row">
             <div class="col-md-6">
-              <div class="standard">
-                <div class="banner"></div>
+                <div class="standard">
+                    <div class="banner"></div>
 {% highlight php %}
 <?php
 
-class Test
+namespace App\DataFixtures\ORM;
+
+use Doctrine\Common\DataFixtures\FixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use App\Entity\User;
+
+class LoadUserData implements FixtureInterface
 {
-    public function test(SomeClass $subject)
+    public function load(ObjectManager $manager)
     {
-        return $subject->doSomething();
+        $user = new User();
+        $user->setUsername('MD');
+        $user->setFullname('Martin Dupond');
+        $user->setBirthday('01/01/1980');
+        $user->setEmail('martin.dupond@mail.com');
+
+        $manager->persist($user);
+        $manager->flush();
     }
 }
 {% endhighlight %}
-              </div>
+                </div>
+
+                <div class="standard">
+{% highlight sh %}
+
+$ php app/console doctrine:fixtures:load
+
+{% endhighlight %}
+                </div>
             </div>
 
             <div class="col-md-6">
-              <div class="rad">
-                <div class="banner"></div>
-{% highlight php %}
-<?php
+                <div class="rad">
+                    <div class="banner"></div>
+{% highlight yml %}
+# Resources/fixtures/orm/users.yml
+
+App\Entity\User:
+    user:
```

Comment:
See my comment above. I would show how easy you can generate multiple users.
`user_{1..10}:`